### DISCUSSION TITLE
DEL move the cursor to the right

### DISCUSSION THREAD

**Discussion by TApplencourt** [ghostty-org/ghostty#3918] [Q&A] [answered]
So it's really weird...

On macOS, after connecting via SSH on a Linux box. Pressing del move the cursor to the right, as if I have pressed space.
Sorry, too lazy to make a video, but it look like

```
$ echo wtf
wtf
```
Now I type `echo wtf` but then presed delete. 
```
$echo wtf   # I pressed Deltete, and the cursor behaved as if I had pressed `space`, but the narrated have been removed
wt
```

I use all the default config and followed the FAQ for `ssh`.

**Comment by TApplencourt** [comment]
Nevermind, `export TERM=xterm-256color` fixed the issue.

**Comment by feekApp** [comment, marked as answer]
Hi,

thats because on your server does not recognize xterm-ghostty,
see: https://ghostty.org/docs/help/terminfo#ssh

use as fallback in your .ssh-config as last lines:
```
# .ssh/config
Host *
  SetEnv TERM=xterm-256color
```

**Reply by TApplencourt** [reply]
Yes of course,

But the doc said "If copying around terminfo entries is untenable" with is wrong, at least in my case. The "Copy Ghostty's terminfo to a remote machine" didn't help. (due to some Intel mac or whatever shaningan)

So the `term=` is the only option not a fall back if I don't want to copy `infocmp`

So maybe my "recommendation" is to update the doc saying "If copying terminfo is untenable or don't solve terminal issuee (del not working, wrong icon...) try..."
